src/components/utils/downloadUtils.ts

```diff
@@ -113,6 +113,7 @@ export function useDownloadUtils() {
                     abortController2
                 );
 
+                // @ts-ignore
```

Comment:
```suggestion
               let downloadFileName =
                    result.headers.get('Content-Disposition')?.split('filename=')[1] ??
                    fileName ??
                    caseElement.elementName;
```
Maybe we can enrich a bit defaulting strategy

Reply:
:+1:   Done